### PR TITLE
Packaging CBZ and commandline arguments

### PR DESCRIPTION
**Pull request by matrix07012**
I added the ability to package chapters into CBZ, commandline arguments, retrying downloads, tweaked README and bumped the version to v0.3, because I think that new features warrant it.

### CODE REVIEW

File: mangadex-dl.py [hunk 1]
```diff
@@ -132,12 +135,17 @@ def dl(manga_id, lang_code, zip_up, tld="org"):
...
-			else:
-				print("Encountered Error {} when downloading.".format(e.code))
+			for _ in range(0,10):
+				try:
+					r = scraper.get(url)
+					if r.status_code == 200:
+						with open(outfile, 'wb') as f:
+							f.write(r.content)
+				except:
+					print("Encountered an error when downloading. Retrying...")
```
Comment (orbea) [inline mangadex-dl.py]: Maybe it would be better to keep the old error, but only print it when bailing out after the last retry?

**Comment by frozenpandaman** [comment]
Hey @matrix07012! I think this functionality would be very cool to have, and making it (and the current language code thing) available via an optional argument with argparse is a great suggestion. Not sure if you've been following the news with the site in the past few months & the leak/hiatus/downtime, but there's an all-new API for the soon-to-be-newly-refreshed site; I've now updated the script (huge revamp) to work with it – and it does, even with the site frontend still down. If you'd be interested in fixing up this PR (I know it's way old now, apologies!) to incorporate this functionality I'd be keen to include it, otherwise I might get around to it some point on my own. :D

Thanks!

**Comment by matrix07012** [comment]
Thanks for a reminder. I updated the branch and also fixed a bug that I discovered where the files in the CBZ retained their entire directory structure.

**Comment by frozenpandaman** [comment]
Thanks, awesome! I'll merge this into a new branch to do some final touchups/changes (I think just having it be an argument rather than a prompt every run will be best, and want to update the readme, etc.) before merging it into master!

**Pull request merged**